dex/src/instruction.rs

```diff
@@ -616,9 +696,13 @@ impl MarketInstruction {
                 let client_id = array_ref![data, 0, 8];
                 MarketInstruction::CancelOrderByClientIdV2NoError(u64::from_le_bytes(*client_id))
             }
-            (20, 46) => MarketInstruction::NewOrderV3NoRent({
```

Comment:
did you mean to remove this `NoRent` ix? also down below `new_order_no_rent` is removed

Reply:
Yeah @xjcaa this was on purpose right?

Reply:
yes